api/lang/completionStep.go

```diff
@@ -43,13 +40,14 @@ func stepCompletion(line, pLine string, params lsp.TextDocumentPositionParams) (
 		cText := prefix + addPlaceHolders(c.StepValue.StepValue, c.StepValue.Parameters)
 		list.Items = append(list.Items, newStepCompletionItem(c.StepValue.ParameterizedStepValue, cText, concept, fText, editRange))
 	}
-	allSteps := append(allUsedStepValues(), allImplementedStepValues()...)
+	s, err := allImplementedStepValues()
```

Comment:
An error is thrown if the runner is not up yet. This change would mean that an error is sent back to the client. So if no runner is started, autocomplete of steps may always throw an error and not  show auto complete steps to the user. I am not sure if this is what is expected here

Reply:
@nehashri - it should be handled via https://github.com/getgauge/gauge/pull/957/commits/b40f5d95610d04fbc41b5d8eefe8b2b072ad06fe#diff-0cdee1df6466adfecc39d07cef7c843fR71